src/general/interface.c

```diff
@@ -432,6 +490,12 @@
 
                     break;
 
+                case interface_pulseaudio:
```

Comment:
In interface_destroy, the deviceName is never freed and I don't understand the comment.

Reply:
At first I didn't figure out the comment either, but then I figured it out.
It's related to this:
https://github.com/introlab/odas/pull/253#discussion_r814044374
It used to copy the pointer in `interface_construct_soundcard_from_name`.
I will uncomment the free in the destructor, and fix `interface_construct_soundcard` which now leaks 1024 bytes every time

Reply:
But I don't think it was ever cleaned up correctly previously anyway, or a least I can't find any `free` on `deviceName` anywhere in ODAS' code. C is great.

Reply:
Even when the deviceName was not copied, it was not cleaned! Me too, I don't like C! RAII is the best!

Reply:
Should be *better* now